pkgs/test/make-binary-wrapper/add-flags/add-flags.cmdline

```diff
@@ -1,4 +1,6 @@
     --append-flags "-foo -bar" \
     --add-flags "-x -y -z" \
     --add-flags -abc \
-    --add-flags "-g *.txt"
+    --add-flags "-g *.txt" \
+    --add-flag 'test var here' \
+    --append-flag 'test var 2 here'
```

Comment:
Let's move the `--add-flag` in between the `--add-flags` to test interspersing.

Also, let's change `*.txt` to `*`, since the point of that is to whether filename expansion is performed and there is more likely going to be *some* file in the current directory than some file *ending in `.txt`*.